packages/components/tabs/src/tab-group.stories.ts

```diff
@@ -138,6 +145,13 @@ export const Links: Story = {
   }
 };
 
+export const NoPanels: Story = {
+  args: {
+    ...Basic.args,
+    tabPanels: undefined
+  }
+};
+
 export const OverflowHorizontal: Story = {
```

Comment:
It looks like the focus ring in the menu button is a little bit clipped on the left?

<img width="154" alt="image" src="https://github.com/user-attachments/assets/98942237-b792-4a7c-a567-e185d69b82f7" />